### PR TITLE
[Change] Added 1.15.2 Support

### PR DESCRIPTION
**Pull request by CDAGaming**

### CODE REVIEW

**Comment by Earthcomputer** [comment]
Pre-release verisons will not be supported. Won't merge this unless you're planning on updating it for the 1.15.2 release.

**Comment by CDAGaming** [comment]
I intend to keep this updated until then hopefully. That, or you'll beat me to it xD

**Comment by CDAGaming** [comment]
Review Re-Requested, as 1.15.2 is now in full release status.

File: src/api/java/net/earthcomputer/multiconnect/api/EnumProtocol.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
     // IMPORTANT: only add to the end of this enum - do not insert in the middle for new versions
     AUTO("Auto", -1),
+    V1_15_2("1.15.2", Protocols.V1_15_2),
```
Comment (Earthcomputer) [inline src/api/java/net/earthcomputer/multiconnect/api/EnumProtocol.java]: Please only add constants to the bottom of this enum

Comment (CDAGaming) [reply src/api/java/net/earthcomputer/multiconnect/api/EnumProtocol.java]: Resolved (RIP OCD Users looking at the code xD)

**Pull request merged**